src/components/repository-browser.less

```diff
@@ -59,6 +59,10 @@
   }
 }
 
+.RepositoryBrowser-tableDeleteCell {
+  color: red;
```

Comment:
We could/should use the defined color ranges we have (`@red-1`, `@red-2` etc)

Reply:
Fixed, thanks. I guess we should make a ticket, we have lots of "red" in the code :))

Reply:
Yeah we need to add an `@error-color: @red-x` that could then also be changed in the system that uses educandu

Reply:
From ant-design we have:

~~~
@primary-color          : @global-primary-color;
@info-color             : @blue-6;
@success-color          : @green-6;
@processing-color       : @blue-6;
@error-color            : @red-6;
@highlight-color        : @red-6;
@warning-color          : @gold-6;
@normal-color           : #d9d9d9;
~~~

but I think it's fine for now (we have to go over the "theming" variables once again when we polish OMA)